<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>569</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>569</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>569</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ataxx</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Ataxx" name="ataxxWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>511</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="newGameButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&amp;New Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadGameButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>390</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&amp;Load Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveGameButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>330</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&amp;Save Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="configButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>450</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&amp;Config</string>
    </property>
   </widget>
   <widget class="QLabel" name="player1Label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Black: </string>
    </property>
   </widget>
   <widget class="QLabel" name="player2Label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>White:  </string>
    </property>
   </widget>
   <widget class="QLabel" name="player1NameLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Player1</string>
    </property>
   </widget>
   <widget class="QLabel" name="player2NameLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Player2</string>
    </property>
   </widget>
   <widget class="QLabel" name="currPlayerLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Player:</string>
    </property>
   </widget>
   <widget class="QLabel" name="currPlayerNameLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>240</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
   </widget>
   <widget class="QLabel" name="player1ScoreLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="player2ScoreLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="player1ScoreValue">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>72</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="player2ScoreValue">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;Menu</string>
    </property>
    <widget class="QMenu" name="menu_Get_Hint">
     <property name="title">
      <string>&amp;Get Hint</string>
     </property>
     <addaction name="actionPlayer_1"/>
     <addaction name="actionPlayer_2"/>
    </widget>
    <addaction name="actionNewGame"/>
    <addaction name="actionSaveGame"/>
    <addaction name="actionLoadLastSave"/>
    <addaction name="separator"/>
    <addaction name="menu_Get_Hint"/>
    <addaction name="actionConfig"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewGame">
   <property name="text">
    <string>&amp;New Game</string>
   </property>
   <property name="toolTip">
    <string>Start New Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionLoadLastSave">
   <property name="text">
    <string>&amp;Load Last Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSaveGame">
   <property name="text">
    <string>&amp;Save Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="text">
    <string>&amp;Config</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionPlayer_1">
   <property name="text">
    <string>Player &amp;1</string>
   </property>
  </action>
  <action name="actionPlayer_2">
   <property name="text">
    <string>Player &amp;2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Ataxx</class>
   <extends>QWidget</extends>
   <header>ataxx.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>newGameButton</tabstop>
  <tabstop>saveGameButton</tabstop>
  <tabstop>loadGameButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
